### PR TITLE
feat(api): env var to control exam environment

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

@ShaunSHamilton I went overboard with caution and made it impossible to turn on the exam env while we're in production mode. I'll probably have to change this as we get closer to deployment.

### CODE REVIEW

File: api/src/app.ts [hunk 1]
```diff
@@ -187,8 +187,7 @@ export const build = async (
     }
   });
 
-  // NOTE: Code behind the `FREECODECAMP_NODE_ENV` var is not ready to be deployed yet.
-  if (FREECODECAMP_NODE_ENV !== 'production') {
+  if (FCC_ENABLE_EXAM_ENVIRONMENT) {
```
Comment (naomi-lgbt) [inline api/src/app.ts:190]:
```suggestion
  if (FCC_ENABLE_EXAM_ENVIRONMENT === "true") {
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: api/src/app.ts (hunk 1 above)
Comment (ojeytonwilliams) [reply api/src/app.ts:190]: It can't be `"true"`, since env converts it to `true`. Booleans are easier to work with, imo.

Comment (ojeytonwilliams) [reply api/src/app.ts:190]: I did completely fail to update the tests, mind. Fixed now.

**Review by naomi-lgbt — APPROVED** [review]

**Review by ShaunSHamilton — APPROVED** [review]
Thanks for this 👍

**Comment by ShaunSHamilton** [comment]
Unrelated test failure.

**Pull request merged**